sql/migrations/20220205202111_world.sql

```diff
@@ -278,6 +278,218 @@ INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3
 INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3`, `value4`, `flags`) VALUES (261, 52, 12478, 12496, 12487, 0, 1);
 
 
+-- 1111:  Not (Source's Entry Is 15882)
+INSERT INTO `conditions` (`condition_entry`, `type`, `value1`, `value2`, `value3`, `value4`, `flags`) VALUES (1111, 16, 15882, 0, 0, 0, 1);
+
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180771, 1111, 0, 5086, 5875);
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180868, 1111, 0, 5086, 5875);
+INSERT INTO `spell_script_target` (`entry`, `type`, `targetEntry`, `conditionId`, `inverseEffectMask`, `build_min`, `build_max`) VALUES (26347, 0, 180850, 1111, 0, 5086, 5875);
```

Comment:
Whats the point of the condition here given that the entry is part of this table? 180850 is not equal to 15882, so what does the condition do?

Reply:
![Firework_explaination](https://user-images.githubusercontent.com/357606/164889036-26d7f8ab-6f9a-427e-b6e0-7af818a7de87.png)